region/lowernorfair/east.json

```diff
@@ -3251,18 +3027,12 @@
                   "heatProof",
                   "canHeatRun"
                 ]},
-                {"roomReset":{
-                  "nodes": [2],
```

Comment:
This one is touchy, but without those Dessgeegas, you can't really line up the iframes properly to charge blue suit without a short charge. It's probably why I put that in.